sccm/core/plan-design/configs/support-for-windows-10.md

```diff
@@ -79,3 +79,6 @@ The following table lists the versions of the Windows 10 ADK that you can use wi
 
  > [!Note]  
  > Configuration Manager only supports x86 and amd64 components of the Windows 10 ADK. It doesn't currently support ARM or ARM64 components. 
+
+# ADK version support for Server OS deployment: 
```

Comment:
Please change from H1 header to a tip, for example: 
`> [!Tip]`

Reply:
title changed